official/resnet/keras/resnet_model.py

```diff
@@ -218,28 +218,30 @@ def ResNet50(include_top=True,
   x = tf.keras.layers.Activation('relu')(x)
   x = tf.keras.layers.MaxPooling2D((3, 3), strides=(2, 2))(x)
 
-  x = conv_block(x, 3, [64, 64, 256], stage=2, block='a', strides=(1, 1))
+  strides = 2
+
+  x = conv_block(x, 3, [64, 64, 256], stage=2, block='a', strides=strides)
   x = identity_block(x, 3, [64, 64, 256], stage=2, block='b')
   x = identity_block(x, 3, [64, 64, 256], stage=2, block='c')
 
-  x = conv_block(x, 3, [128, 128, 512], stage=3, block='a')
+  x = conv_block(x, 3, [128, 128, 512], stage=3, block='a',strides=strides)
```

Comment:
conv_block already has a default strides=(2, 2). I think the change you are looking to make is inside conv_block. From Toby's doc I understand we want to change the strides for the first 3X3 convolution in each block.
Each conv_block  has 4 Conv2D layer calls. We want to change the strides for the second Conv2D layer with a kernel size of 3.
* We have a stride (2, 2) for the first conv2d layer. set this to the default of (1, 1). 
* Set the strides argument of the second Conv2D layer of the block to (2, 2) (or in this case the strides parameter of the function). 
I don't think you need to change the pool size of the AveragePooling layer. Can you try this out? Thanks!

Reply:
Done.